### ISSUE TITLE
ip-add fails when accessing CLIArg.path

### ISSUE THREAD

**Issue by asauber** [linode/linode-cli#27]
Executing `linode-cli linode ip-add` fails with the following traceback.

```
asauber@linaret manager # linode-cli linodes ip-add
Traceback (most recent call last):
  File "/usr/local/bin/linode-cli", line 11, in <module>
    sys.exit(main())
  File "/Library/Python/2.7/site-packages/linodecli/__init__.py", line 209, in main
    cli.handle_command(parsed.command, parsed.action, args)
  File "/Library/Python/2.7/site-packages/linodecli/cli.py", line 413, in handle_command
    result = self.do_request(operation, args)
  File "/Library/Python/2.7/site-packages/linodecli/cli.py", line 323, in do_request
    parsed_args = operation.parse_args(args)
  File "/Library/Python/2.7/site-packages/linodecli/operation.py", line 111, in parse_args
    print(arg.path)
AttributeError: CLIArg instance has no attribute 'path'
```

With a bit of debugging, it was found that the offending `CLIArg` was `type`, which did have a `name` and `description`, but no `path`.

**Comment by Dorthu** [comment]
I can reproduce this, looks like python2 has an issue somewhere.  I'm looking into it.

**Comment by snitko** [comment]
Thanks, using python3 and pip3 works.

**Comment by Dorthu** [comment]
This should be fixed in `2.0.1b0`
